web/sites/default/config/views.view.manage_group_content_revision.yml

```diff
@@ -3335,22 +3335,28 @@ display:
               action_id: 'entity:break_lock:node'
               preconfiguration:
                 label_override: 'Break lock'
-            11:
-              action_id: mark_for_review_action
-              preconfiguration:
-                label_override: 'Mark for Review'
```

Comment:
@radmosley, why did you remove Mark for Review?

Reply:
Ah... my bad. I swear I didn't see it there when I was reviewing earlier. I have reviewed way too much code today. 😅